cmd/root.go

```diff
@@ -79,9 +77,23 @@ func getLatestReleaseInfo(info chan<- string) {
 
 func getOcClient() *occlient.Client {
 	client, err := occlient.New()
+	checkError(err, "")
+	return client
+}
+
+// checkError prints the cause of the given error and exits the code with an
+// exit code of 1.
+// If the context is provided, then that is printed, if not, then the cause is
+// detected using errors.Cause(err)
+func checkError(err error, context string) {
 	if err != nil {
-		fmt.Println(err)
+		log.Debugf("Checking for error:\n%v", err)
```

Comment:
I'm not sure if "Checking for error" is accurate here.
This is in if statement `err!=nil` so when it hits this line we already know that there has been error.

Reply:
changed